README.adoc

```diff
@@ -229,12 +242,31 @@ message in the browser. Otherwise, this method returns the cached property value
 You successfully set up your microservice to have fault tolerance capability.
 
 
+== Enabling metrics for the fault tolerance methods
+
+server.xml
+[source, xml, linenums, role='code_column']
+----
+include::finish/src/main/liberty/config/server.xml[]
+----
+
+MicroProfile Fault Tolerance integrates with MicroProfile Metrics to provide metrics for the annotated fault tolerance methods.
+When both the [hotspot=mpFaultTolerance]`mpFaultTolerance` and the [hotspot=mpMetrics]`mpMetrics` features are included in the [hotspot]`server.xml` configuration file,
+the `@Fallback` fault tolerance annotation supplies metrics that count the total number of annotated method invocations and failed invocations,
+as well, the total number of the fallback method calls.
+
+Note that the [hotspot=mpMetrics]`mpMetrics` feature requires SSL and the configuration has been provided for you. The [hotspot=quickStartSecurity]`quickStartSecurity` and [hotspot=keyStore]`keyStore` configuration elements provide basic security to secure the
+server. When you visit the `/metrics` endpoint, use the credentials defined in the server configuration to log in to view the data for the fault tolerance methods.
+
+You can learn more about MicroProfile Metrics in the https://openliberty.io/guides/microprofile-metrics.html[Providing metrics from a microservice^] guide. You can also learn more about the MicroProfile Fault Tolerance and Metrics integration in the https://github.com/eclipse/microprofile-fault-tolerance/releases[MicroProfile Fault Tolerance specification^].
+
 // =================================================================================================
 // Building and running the application
 // =================================================================================================
 
-[role='command']
-include::{common-includes}/mvnbuild.adoc[]
+== Building and running the application
+
+The Open Liberty server was started in development mode at the beginning of the guide and all the changes were automatically picked up.
```

Comment:
Change to:
"The Open Liberty server started in development mode at the beginning of the guide and all the changes were automatically picked up."

Reply:
Updated